modules/cyclone-dx/src/test/resources/logback.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+  ~ Copyright (c) Robert Bosch Manufacturing Solutions GmbH 2020.
+  ~
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v2.0
+  ~ which accompanies this distribution, and is available at
+  ~ http://www.eclipse.org/legal/epl-v20.html
+  ~
+  ~ SPDX-License-Identifier: EPL-2.0
+  -->
+<configuration>
+
+   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+      <encoder>
+         <pattern>%d{YYYY-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+      </encoder>
+   </appender>
+   
+   <logger name="com.bosch.bios.bot" level="ERROR" />
```

Comment:
This name is wrong. com.bosch is not applicable to Open Source.

Reply:
Good catch. See finding above. Should be resolved by that.